lib/json_schema_form/umm_form.rb

```diff
@@ -298,6 +298,51 @@ def accordion_id
       top_key.underscore.dasherize
     end
   end
+
+  def field_title_not_accordion_title?
+    accordion_titles = Array.new
+    json_form.parsed_json['forms'].each do |form|
+      form['items'].each { |accordion| accordion_titles << accordion['title'] }
+    end
+
+    accordion_titles.none? { |accordion_title| accordion_title == title }
+  end
+
+  def label_id
+    value = parsed_json.fetch('key','').split('/').first
+    value = parsed_json.fetch('key','').split('/')[1] if value.blank?
+
+    # note: field != form != accordion in the context of the below comment:
+    # value == top_key: checks if the label is a top-level field (we don't want to put label id's on fields that aren't
+      # linked via progress circle)
+    # !parsed_json['noLabel']: there are no progress circles that link to fields with a truthy 'noLabel' key.
+    # field_title_differs_from_accordion_title?: makes sure the fields that share the same title as their accordion don't
```

Comment:
Minor comment error. Still mentions old method. Should me field_title_not_accordion_title.

Reply:
good catch.